### PR TITLE
Format and getAge methods added 

### LINKED ISSUE #3: Add formating
See package spec https://github.com/personnummer/meta

### PR DESCRIPTION
**Pull request by VoxSecundus**
Fixes #3

### CODE REVIEW

**Comment by frozzare** [comment]
Looks good, could you add some unit tests to test format method? You could look at how we do it the JavaScript repository. https://github.com/personnummer/js/blob/master/test.js#L63-L87

File: lib/personnummer.rb [hunk 1]
```diff
@@ -62,4 +62,48 @@ def self.valid(str)
...
+          d = Time.new
+          baseYear = 0
+
+          if sep == '+'
+            baseYear = d.year - 100
+          else
+            baseYear = d.year
+          end
+          century = "#{(baseYear - ((baseYear - year.to_i) % 100)).digits[-2..-1].reverse.join('')}"
+        end
```
Comment (frozzare) [inline lib/personnummer.rb]: Row 71-101 should be moved to a own method so we can use the same logic in other methods, e.g `getAge` 

In the other repositories we have a function/method called `getParts` , so something like that :) https://github.com/personnummer/js/blob/master/src/personnummer.js#L48-L96

**Comment by frozzare** [comment]
Travis tests failed du to undefined method:

```
NoMethodError: undefined method `digits' for 1964:Fixnum
    /home/travis/build/personnummer/ruby/lib/personnummer.rb:100:in `format'
```

You can ignore the test for `Ruby: jruby-9.0.4.0` since it's another problem and not your code.

File: lib/personnummer.rb (hunk 1 above)
Comment (VoxSecundus) [reply lib/personnummer.rb]: Did consider doing that, thought I'd wait and see what you said first. On it now :)

**Comment by frozzare** [comment]
Still, we have to test `getAge` method to know that the method are working the way it should. You could use this test cases https://github.com/personnummer/js/blob/master/test.js#L88

**Comment by VoxSecundus** [comment]
This PR is now out of date and irrelevant.

**Pull request closed without merging**